src/noteburst/handlers/prototyping/handlers.py

```diff
@@ -174,3 +175,32 @@ async def run_code(
         logger.info("Stopping JupyterLab")
         await jupyter_client.stop_lab()
         logger.info("Stopped JupyterLab")
+
+
+@prototype_router.post(
+    "/ping", description="Enqueue the ping worker task.", status_code=202
+)
+async def post_ping(
+    *,
+    request: Request,
+    logger: structlog.BoundLogger = Depends(logger_dependency),
+    arq_queue: ArqQueue = Depends(arq_dependency),
+) -> QueuedJob:
+    logger.info("Enqueing a ping task")
+    job_metadata = await arq_queue.enqueue("ping")
+    logger.info("Finished enqueing a ping task", job_id=job_metadata.id)
+    return await QueuedJob.from_job_metadata(job=job_metadata, request=request)
```

Comment:
Consider returning a redirect to `/jobs/{job_id}` here instead. I tend to like that pattern better than the POST returning the metadata directly, both because the metadata code is only in one place and because it reveals the URL for the job to a REST client.

Reply:
Yes, I do intend to set a Location header to the job endpoint, just didn't get to it for this demo.

Sending a Redirect is an interesting idea.